mcode/pims
# Pims auth number for etasu

## Describe your changes

Get auth number from the newRx and use that for making the etasu call if there is an auth number.

To Test:
Go through workflow as expected. Send rx to pharmacy after filling out all forms so auth number exists. Go to pharmacy and verify the status. This should use the auth number to get the etasu. If auth number is blank, use the patient and medication to get etasu.

## Issue ticket number and Jira link

https://jira.mitre.org/browse/REMS-673 

## Checklist before requesting a review

## Checklist for conducting a review

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.
